src/pages/basic_data/app_admin/NviAdminStatusPage.tsx

```
@@ -11,91 +12,28 @@ import {
-      approved: 5,
-      rejected: 1,
-      disputes: 0,
-      total: 6,
-    },
-  ]
-    .filter((obj) => selectedSector === null || obj.sector === selectedSector)
-    .filter((obj) => {
-      if (institutionSearch === null) {
-        return true;
-      }
+  const filteredData = (reportsQuery.data?.institutions ? [...reportsQuery.data?.institutions] : [])
```

Biome (2.4.4)

[error] Unsafe usage of optional chaining.

(lint/correctness/noUnsafeOptionalChaining)